src/vfc_precexp

```diff
@@ -63,7 +63,11 @@ def shell(cmd, newEnv, maxTimeout=None):
                             stdout=subprocess.PIPE,
                             env=env)
 
-    proc.wait(maxTimeout)
+    try:
+        proc.wait(maxTimeout)
+    except subprocess.TimeoutExpired:
+        proc.terminate()
+        return 42
```

Comment:
Why 42 here ?

Reply:
Missed that one... ;) 
returncode¶
Exit status of the child process. Typically, an exit status of 0 indicates that it ran successfully.

A negative value -N indicates that the child was terminated by signal N (POSIX only).

Since you call terminate, sigtermcode would probably be the best choice